### PR TITLE
vsphere disconnected techpreview

### PR DESCRIPTION
**Pull request by huali9**
remove `cucushift-installer-rehearse-vsphere-ipi-capi` workflow and add `cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview` workflow. as now we change to use one workflow to cover both capi and ccm.
@sunzhaohua2  @jhou1 @miyadav Please help to take a look.

### CODE REVIEW

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ tests:
   interval: 960h
   steps:
     cluster_profile: vsphere
-    workflow: cucushift-installer-rehearse-vsphere-ipi-capi
+    workflow: cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview
```
Comment (jinyunma) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml:59]: For disconnected scenario, we need to use cluster profile "vsphere-discon", which will choose correct disconnected network for cluster installation on vSphere, suggest to add new section here which is the same as one in above 4.10 job.  https://github.com/openshift/release/pull/29844/files#diff-ea05105b2925414db38c1251c56f44730fa9183edb850cef6d8b6fddb53f1a92R74-R78

Comment (huali9) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml]: Got it, thanks! as we already validate it in 4.10 job, so we don't need to revalidate it in 4.11 as they are the same for the two versions, so change it to `cucushift-installer-rehearse-vsphere-ipi` as `cucushift-installer-rehearse-vsphere-ipi-capi` workflow does not exist.

**Comment by huali9** [comment]
The new workflow `cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview` validate passed in 4.10 job:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29844/rehearse-29844-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.10-installer-rehearse-vsphere-discon/1541606185008893952
4.11 job failed but the workflow `cucushift-installer-rehearse-vsphere-ipi` is an old one, so maybe it's flake? Can we ignore the error and continue merging this pr? @jinyunma

**Comment by jinyunma** [comment]
> The new workflow `cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview` validate passed in 4.10 job: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29844/rehearse-29844-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.10-installer-rehearse-vsphere-discon/1541606185008893952 4.11 job failed but the workflow `cucushift-installer-rehearse-vsphere-ipi` is an old one, so maybe it's flake? Can we ignore the error and continue merging this pr? @jinyunma


checked that one master node was not ready, looks like a flake.

**Comment by huali9** [comment]
all tests passed!
@sunzhaohua2 @jhou1 @miyadav @jianlinliu PTAL, thanks!

**Pull request merged**